fft/src/KokkosFFT_ROCM_plans.hpp

```diff
@@ -159,56 +162,46 @@ auto create_plan(const ExecutionSpace& exec_space,
 
   // Create a plan
   plan   = std::make_unique<PlanType>();
-  status = rocfft_plan_create(&(*plan), place, fft_direction, precision,
+  status = rocfft_plan_create(&(plan->plan()), place, fft_direction, precision,
                               reversed_fft_extents.size(),  // Dimension
                               reversed_fft_extents.data(),  // Lengths
-                              howmany,     // Number of transforms
-                              description  // Description
+                              howmany,      // Number of transforms
+                              *description  // Description
```

Comment:
Why not making this part of the constructor of `ScopedRocfftPlan` ?

Reply:
I moved the logic into the constructor